src/arch/freeRTOS/forte_Init.cpp

```diff
@@ -81,7 +81,7 @@ int forteStartInstanceGeneric(int paArgc, char *paArgv[], TForteInstance* paResu
   const char *pIpPort = parseCommandLineArguments(paArgc, paArgv);
   if((0 != strlen(pIpPort)) && (nullptr != strchr(pIpPort, ':'))) {
     C4diacFORTEInstance *instance = new C4diacFORTEInstance();
-    if(!instance->startupNewDevice(ipPort)) {
+    if(!instance->startupNewDevice(pIpPort)) {
```

Comment:
according to the coding guidelines, type prefixes like `p` are obsolete and shall not be used anymore.
Please fix this by renaming the variable to `ipPort` in lines 81, 82 and 84